pallet/message-transact/src/lib.rs

```diff
@@ -148,6 +148,16 @@ pub mod pallet {
 			};
 
 			let transaction_data: TransactionData = (&*transaction).into();
+			let (weight_limit, proof_size_base_cost) =
```

Comment:
It seems that this implementation has been copied from the upstream source.

Reply:
It would be better to link the source code here.

Reply:
Yes. I'll dig into deep to see if there is a way to reuse upstream function. Keep it now.

Reply:
The current way is fine with me. This piece of code is not like const pallet name, etc. which are tightly bound to something.

Reply:
Otherwise, I need an issue to keep track of this and we can update this in the future. Currently, my memory is not very reliable.

Reply:
#1206